src/verify.js

```diff
@@ -0,0 +1,58 @@
+'use strict';
+const childProcess = require('child_process');
+const debug = require('debug');
+const fs = require('fs');
+
+const log = debug('verify');
+const error = debug('verify:ERROR');
+// For some reason RED is 1.
+error.color = 1;
+
+function isBaselineAllPass(baselineFile) {
+  const lines = fs.readFileSync(baselineFile, 'utf-8').split(/\r?\n/);
+  let allPass = true;
+  lines.forEach((line) => {
+    if (line.startsWith('FAIL') || line.startsWith('WARN')) {
+      allPass = false;
```

Comment:
Hmmm I was confused how to do that since the return would just return within the closure, not the outer function. Right? Unless you meant to add a return in the closure. I could also change this to a normal for loop that way I could return from the outer function.

Reply:
Yeah, you're right, doesn't apply to the closure. Normal for-loop with an early return would just be a micro-optimization, so might not be worth the effort. Feel free to ignore.